project/src/moks/const.ts

```diff
@@ -0,0 +1,61 @@
+import {LivePlaceProps} from '../types/live-place';
+
+const PlaceType: {APARTMENT: string, PRIVATE_ROOM: string} = {
+  APARTMENT: 'Apartment',
+  PRIVATE_ROOM: 'Private room',
+};
```

Comment:
```suggestion
enum PlaceType {
  Apartment = 'Apartment',
  PrivateRoom = 'Private room',
};
```
Мувни это в файл констант

И подправь тип оффера

```
placeType: PlaceType
```